src/services/supabase.ts

```diff
@@ -107,3 +107,59 @@ export const getUserDetails = (user: User | null) => {
         avatar,
     };
 };
+
```

Comment:
Helper function to handle making calls to supabase that don't use the `useSWR` stuff